webrtc.html

```diff
@@ -2763,6 +2763,14 @@
               bandwidth.
             </p>
           </dd>
+
+          <dt>unsigned long maxBitrate</dt>
+          <dd>
+            <p>
+              Indicates the maximum bitrate that can used to send this
```

Comment:
Fixed the typo, and added a comment about how per-transport or per-session bandwidth limits may further constrain the encoding.